modules/37_industry/subsectors/equations.gms

```diff
@@ -12,12 +12,26 @@ q37_demFeIndst(ttot,regi,entyFe,emiMkt)$(    ttot.val ge cm_startyear
   sum(se2fe(entySE,entyFE,te),
     vm_demFEsector(ttot,regi,entySE,entyFE,"indst",emiMkt)
   )
+*** substract chemical feedstocks which are supplied by vm_demFENonEnergySector (see q37_demFeFeedstockChemIndst)
```

Comment:
I would split this into two equations, if not you would allow non energy demand to compensate energy demand, and vice versa depending of the costs in the model. If you want to set the non energy pathway as the upper bound of non nergy use, just use a inequality instead in the second equation in this case. 

(1) FE balance:
vm_demFEsector + vm_demFENonEnergySector =e= vm_cesIO(ttot,regi,in);

(2) non energy pathway
vm_demFENonEnergySector =e= vm_cesIO(ttot,regi,in) * p37_chemicals_feedstock_share(ttot,regi);